### ISSUE TITLE
linter: `jsx-a11y/no-distracting-elements` rule is missing a config option

### ISSUE THREAD

**Issue by connorshea** [oxc-project/oxc#18871]
### What version of Oxlint are you using?

1.43.0

### What command did you run?

`oxlint`

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "plugins": ["jsx-a11y"],
  "rules": {
    "jsx-a11y/no-distracting-elements": [
      "error",
      {
        "elements": ["marquee", "blink"]
      }
    ],
  }
}
```

### What happened?

The option basically just exists to allow you to turn off one of the two default elements that are detected as problematic. It's a bit goofy and you cannot add any _extra_ elements. You can only set which of the two elements to check for.

https://oxc.rs/docs/guide/usage/linter/rules/jsx_a11y/no-distracting-elements.html

Original rule docs:

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-distracting-elements.md

We could potentially consider this an unsupported behavior, but it should be clearly noted in the docs as such if we want to go that route. 

We should also consider modifying this particular config in @oxlint/migrate to remove this option if it is found, and print a warning for the user saying as much.

I ran into this because the airbnb eslint config sets this option to a value explicitly (it gets set to the default value), and so I get an invalid config error in oxlint as a result of migrating. https://github.com/airbnb/javascript/blob/0e2ef178a26ba9ac3495402a182891ad8096d3a0/packages/eslint-config-airbnb/rules/react-a11y.js#L158
